### ISSUE TITLE
Position of fields

### ISSUE THREAD

**Issue by dschrempf** [nextcloud/contacts#3283]
### Is your feature request related to a problem? Please describe.

Hi! I am sure this has come up already, so please feel free to close this issue and refer to the proper place where this is discussed.

At the moment, the positions of the fields in the contact form change seemingly randomly with respect to the number of fields present, and also with respect to the monitor size.

### Describe the solution you'd like

In my opinion, the position of fields should be constant. For example, the position of the Birthday fields should not jump around depending on which other fields are present. In the following screenshots, we see that the Birthday field jumps around like a jumping spider and other fields happily play along the "I appear always where you do not expect me" game.

![image](https://user-images.githubusercontent.com/5596239/227174117-c6be76e8-d399-48e2-887b-a6fccd85b7cd.png)

![image](https://user-images.githubusercontent.com/5596239/227174260-4b82a61c-dc63-40bd-9da4-e7060c88dd35.png)

Thanks for considering this feature request!

**Comment by ChristophWurst** [comment]
https://github.com/nextcloud/contacts/issues/1923

**Comment by arnowelzel** [comment]
https://github.com/nextcloud/contacts/issues/2699 - here you also find a set of CSS rules you can use with the CustomCSS app to change the layout and fix some of the issues.

**Comment by beedell-roke** [comment]
@dschrempf, this appears to have been remediated:

<p></p><img width="1276" height="716" alt="Image" src="https://github.com/user-attachments/assets/a0a45cc3-9f5c-4b83-889b-19ce19387045" /><p></p>

**Comment by dschrempf** [comment]
Right! In general, the Contacts app seems much user-friendly now. Thank you!
